cookbooks/bcpc-hadoop/templates/default/ooz_oozie-env.sh.erb

```diff
@@ -9,19 +9,21 @@
 <% if @ooz_hosts.length >= 2 %>
 export OOZIE_HTTP_HOSTNAME=<%=node[:bcpc][:management][:vip]%>
 <% end %>
-export OOZIE_CONFIG="/etc/oozie/conf"
-export OOZIE_DATA="/var/lib/oozie"
-export OOZIE_LOG="/var/log/oozie"
-export OOZIE_LOG_DIR="/var/log/oozie"
-export OOZIE_PID_DIR="/var/run/oozie"
-export OOZIE_CATALINA_HOME="/usr/lib/bigtop-tomcat"
-export CATALINA_TMPDIR="/tmp"
-export CATALINA_PID="/var/run/oozie/oozie.pid"
-export CATALINA_BASE="/usr/hdp/<%= node[:bcpc][:hadoop][:distribution][:active_release] %>/oozie/oozie-server/"
+export OOZIE_CONFIG=/etc/oozie/conf
+export OOZIE_DATA=/var/lib/oozie
+export OOZIE_LOG=/var/log/oozie
+export OOZIE_LOG_DIR=/var/log/oozie
+export OOZIE_PID_DIR=/var/run/oozie
+export OOZIE_CATALINA_HOME=/usr/lib/bigtop-tomcat
+export CATALINA_TMPDIR=/tmp
+export CATALINA_PID=/var/run/oozie/oozie.pid
+export CATALINA_BASE=/usr/hdp/<%= node[:bcpc][:hadoop][:distribution][:active_release] %>/oozie/oozie-server/
```

Comment:
The quotes were removed just to match `Oozie` [documentation](https://oozie.apache.org/docs/4.0.1/AG_Install.html).